### PR TITLE
Integrate with neighbourhood on PerspectiveHandle

### PR DESCRIPTION
**Pull request by jdeepee**
Depends on https://github.com/perspect3vism/ad4m/pull/13 being merged and published for proper release.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
   },
   "dependencies": {
     "@holochain/conductor-api": "0.0.2",
-    "@perspect3vism/ad4m": "0.1.4",
+    "@perspect3vism/ad4m": "file:../ad4m",
```
Comment (lucksus) [inline package.json:63]: Local path needs changing before merge

File: src/core/PerspectivesController.ts [hunk 2]
```diff
@@ -24,28 +24,14 @@ export default class PerspectivesController {
...
-                this.#perspectiveHandles.set(k, fileObject[k])
-                if (neighbourhoodsFileObject) {
-                    //@ts-ignore
-                    let neighbourhood = neighbourhoodsFileObject[k];
-                    if (neighbourhood != {}) {
-                        let neighbourhoodTyped = neighbourhood as Neighbourhood;
-                        console.debug(`Found neighbourhood for perspective "${k}":`, neighbourhoodTyped)
-                        this.#perspectiveInstances.set(k, new Perspective(fileObject[k], this.#context, neighbourhoodTyped))
-                    }
-                }
```
Comment (lucksus) [inline src/core/PerspectivesController.ts:48]: Is this correct? You're removing the code that was recently added..

Comment (jdeepee) [reply src/core/PerspectivesController.ts:48]: Yep having neighbourhood on the PerspectiveHandle means we dont need another .json file to store neighbourhood data as it is done on the PerspectiveHandle objects already

Comment (lucksus) [reply src/core/PerspectivesController.ts:48]: ah, perfect

File: src/tests/neighbourhood.ts [hunk 3]
```diff
@@ -33,8 +34,12 @@ export default function neighbourhoodTests(testContext: TestContext) {
                 //Check that we got an ad4m url back
                 expect(publishPerspective.split("://").length).toBe(2);
 
+                const perspective = await ad4mClient.perspective.byUUID(create.uuid);
+                expect(perspective?.neighbourhood).toBeDefined();
+
                 const join = await ad4mClient.neighbourhood.joinFromUrl(publishPerspective );
                 expect(join.sharedUrl).toBe(publishPerspective);
+                expect(join.neighbourhood).toBeDefined();
```
Comment (lucksus) [inline src/tests/neighbourhood.ts:42]: Should we also check the neighbourhood's fields (`linkLanguage` and `meta`)?

**Review by lucksus — APPROVED** [review]
Maybe needs one or two more assertions in the test as mentioned above, but looks good to merge.

File: src/tests/neighbourhood.ts (hunk 3 above)
Comment (jdeepee) [reply src/tests/neighbourhood.ts:42]: Sure adding and then merging

**Review by lucksus — APPROVED** [review]

**Pull request merged**